### PR TITLE
renderer fix and bilinear filtering

### PR DESCRIPTION
**Pull request by Kubsy**
Branch by Monty

### CODE REVIEW

File: TombEngine/Shaders/DX11_Rooms.fx [hunk 1]
```diff
@@ -188,8 +188,11 @@ PixelShaderOutput PS(PixelShaderInput input)
 		float4(0.0f, 0.0f, 0.0f, 0.0f);
 
 	lighting -= float3(input.FogBulbs.w, input.FogBulbs.w, input.FogBulbs.w);
-	lighting = saturate(lighting);
+	//if (input.FogBulbs.w > 0.0f)
```
Comment (Lwmte) [inline TombEngine/Shaders/DX11_Rooms.fx:191]: Do not comment out lines like that.

File: TombEngine/Shaders/DX11_Rooms.fx [hunk 2]
```diff
@@ -188,8 +188,11 @@ PixelShaderOutput PS(PixelShaderInput input)
 		float4(0.0f, 0.0f, 0.0f, 0.0f);
 
 	lighting -= float3(input.FogBulbs.w, input.FogBulbs.w, input.FogBulbs.w);
-	lighting = saturate(lighting);
+	//if (input.FogBulbs.w > 0.0f)
+	//	lighting = saturate(lighting);
 	output.Color.xyz = output.Color.xyz * lighting;
+	//if (input.FogBulbs.w == 0.0f)
```
Comment (Lwmte) [inline TombEngine/Shaders/DX11_Rooms.fx:194]: Do not comment out lines like that.

**Review by Lwmte — CHANGES_REQUESTED** [review]

File: TombEngine/Shaders/DX11_Rooms.fx (hunk 1 above)
Comment (MontyTRC89) [reply TombEngine/Shaders/DX11_Rooms.fx]: Done

File: TombEngine/Shaders/DX11_Rooms.fx (hunk 2 above)
Comment (MontyTRC89) [reply TombEngine/Shaders/DX11_Rooms.fx]: Done

**Pull request merged**